2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,33 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+const logRecipe = (recipe) => {
+  console.log(recipe.title);
+  console.log(`Serves: ${recipe.servings}`);
+  console.log("Ingredients:");
+  recipe.ingredients.forEach((ingredient) => console.log(ingredient));
+}
```

Comment:
I like how you've extracted all your console.logs needed into one function- this works really well & logs exactly as stated in the question, well done